src/riderStatus.js

```diff
@@ -1,10 +1,8 @@
 /* eslint-disable indent */
 import protobuf from 'protobufjs';
 
-const extraVariables = ['roadID', 'rideOns', 'isTurning', 'isForward', 'cadence'];
-
-const proto = `syntax=\"proto3\";
-    message Status {
+const proto = `syntax="proto3";
```

Comment:
I think creating zwiftProtobuf.js is a good idea. I thought about doing that but figured I didn't want to change code structure in a fork.

I don't have a strong opinion about whether the transpiling is better or worse, since I'm pretty new to node.js. I like the ES2015 features, but as you say many of them are already present in Node.js. It would be convenient to be able to import subsets of the API, though.

I'll publish my current version of the telemetry app later today as an example of how I'm using the API for UDP decoding. That might help give some context.